__tests__/app.test.js

```diff
@@ -456,3 +456,57 @@ describe("DELETE /api/comments/:comment_id", () => {
       });
   });
 });
+
+describe("GET /api/users", () => {
+  test("responds with status code 200", () => {
+    return request(app).get("/api/users").expect(200);
+  });
+  test("responds with an array of user objects with username, name and avatar_url properties", () => {
+    return request(app)
+      .get("/api/users")
+      .then((response) => {
+        const { users } = response.body;
+        expect(users.every((user) => user.hasOwnProperty("username"))).toBe(
+          true
+        );
+        expect(users.every((user) => user.hasOwnProperty("name"))).toBe(true);
+        expect(users.every((user) => user.hasOwnProperty("avatar_url"))).toBe(
+          true
```

Comment:
this is a lot of work 

You might want to consider using another jest matcher that might make these assertions a bit nicer  - toMatchObject is pretty good https://jestjs.io/docs/expect#tomatchobjectobject 

This would allow you to check the type of the value on each property too